### PR TITLE
Corrected script for some item

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: #7073

* **Server Mode**: re

* **Description of Pull Request**: 
Thanks to @FrFrMako

### CODE REVIEW

File: db/re/item_combos.yml [hunk 1]
```diff
@@ -1061,37 +1075,57 @@ Body:
...
+      - Combo:
+          - Memorize_Book_    # 2121
+          - One_Eyed_Glass    # 2239
+          - Pocket_Watch_    # 2717
+          - Pocket_Watch__    # 2971
+      - Combo:
+          - Memorize_Book_    # 2121
+          - One_Eyed_Glass_    # 18874
+          - Pocket_Watch_    # 2717
+          - Pocket_Watch__    # 2971
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_combos.yml]: sorry, i was mistaken

File: db/re/item_combos.yml [hunk 2]
```diff
@@ -355,17 +355,31 @@ Body:
...
+      - Combo:
+          - Principles_Of_Magic    # 1572
+          - Librarian_Glove    # 2716
+          - Pocket_Watch_    # 2717
+      - Combo:
+          - Principles_Of_Magic    # 1572
+          - Librarian_Glove    # 2716
+          - Pocket_Watch_    # 2717
+          - Pocket_Watch__    # 2971
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_combos.yml]: sorry, i was mistaken

File: db/re/item_combos.yml [hunk 3]
```diff
@@ -355,17 +355,31 @@ Body:
...
-          - Librarian_Glove
-          - Pocket_Watch
+          - Ancient_Magic    # 1573
+          - Librarian_Glove    # 2716
+          - Pocket_Watch_    # 2717
+      - Combo:
+          - Ancient_Magic    # 1573
+          - Librarian_Glove    # 2716
+          - Pocket_Watch_    # 2717
+          - Pocket_Watch__    # 2971
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_combos.yml]: sorry, i was mistaken

File: db/re/item_combos.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_combos.yml]: when equip (Pocket_Watch_    # 2717) and (Pocket_Watch__    # 2971) Combos bonus script calculated 2 times.

[the remark this answers is not in the document]
Comment (Atemo) [reply db/re/item_combos.yml]: Ha, any idea how it behaves on official servers?

[the remark this answers is not in the document]
Comment (Atemo) [reply db/re/item_combos.yml]: Let's leave it like that for now

**Comment by Atemo** [comment]
Thanks again!

**Pull request merged**